tests/pkcs1.rs

```diff
@@ -169,29 +169,33 @@ fn decode_rsa4096_pub_pem() {
 fn encode_rsa2048_priv_pem() {
     let key = RsaPrivateKey::from_pkcs1_pem(RSA_2048_PRIV_PEM).unwrap();
     let pem = key.to_pkcs1_pem(Default::default()).unwrap();
```

Comment:
It's inserting the `\r` because `Default::default()` is actually `LineEnding::default()`, which uses OS-specific line endings.

To make the test always use `\n` line endings regardless of OS, you can use:

```suggestion
    let pem = key.to_pkcs1_pem(LineEnding::LF).unwrap();
```